README.md

```diff
@@ -1 +1,23 @@
-# parasoft-demo-app-mobile
\ No newline at end of file
+# parasoft-demo-app-mobile
+The Parasoft Demo Application is an example Android application. The application is configurable, and is used to demonstrate functionality in a variety of Parasoft tools.
+
+## Getting Started
+### Create emulator
+1. Open **Device Manager**, click **Create device**.
+2. Choose a device which has the resolution higher than **720x1280**, we recommend the device **Pixel 5 (1080x2340)**.
+3. Click **next**, choose an API version which **higher or equal to 32**, then click finish.
+
+### Build apk
+Select the device that you just created, and click **Run 'app'**, Android stdio will build and open the app automatically.
+
+## Using the Demo Application
+1. Make sure the PDA service is running
```

Comment:
Use full app name instead of PDA

Reply:
Add the service project link https://github.com/parasoft/parasoft-demo-app. Like
Make sure the [parasoft-demo-app](https://github.com/parasoft/parasoft-demo-app) service is running.